database.js

```diff
@@ -0,0 +1,28 @@
+const { Client } = require("pg");
+require('dotenv').config()
+
+const client = new Client({
+  host: process.env.HOST,
+  port: process.env.PORT,
+  user: process.env.USERNAME,
+  password: process.env.PASSWORD,
+  database: process.env.DATABASE 
+});
+
+client.connect().then(() => {
+  createPgAccount("username", "password", "database");
```

Comment:
what is this doing.

Reply:
calling the function createPgAccount and passing in dummy data for now. Eventually, data from the sign-up page will get passed in.

Reply:
remove, or else this script will throw error.